quickshop-bukkit/src/main/java/com/ghostchu/quickshop/localization/text/SimpleTextManager.java

```diff
@@ -454,7 +455,11 @@ public ProxiedLocale findRelativeLanguages(@Nullable final String langCode) {
       Log.debug(Level.WARNING, "Failed to solve the player locale: " + locale + ": " + e.getMessage());
     }
 
-    return new ProxiedLocale(langCode, result, NumberFormat.getCompactNumberInstance(locale, NumberFormat.Style.SHORT), locale);
+    return new ProxiedLocale(langCode, result, getCompactNumberInstance(locale), locale);
+  }
+
+  private NumberFormat getCompactNumberInstance(@NotNull final Locale locale) {
+        return numberFormatCache.computeIfAbsent(locale, l->NumberFormat.getCompactNumberInstance(l, NumberFormat.Style.SHORT));
```

Comment:
Should have blank line above here per the updated style guidelines.

Reply:
fixed